### PR TITLE
fix(chat): reset emotes search on modal close/change

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
reset emotes search on modal close/change

**Which issue(s) this PR fixes** 🔨
[AP-1118](https://satellite-im.atlassian.net/browse/AP-1118)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by phillsatellite** [comment]
When tested with Netlify I noticed Search works for Glyphs and no longer works for emoji's.

https://user-images.githubusercontent.com/93608357/159575578-da0c4946-ce45-4a56-9eba-7ed404f86fcd.mov

File: components/views/chat/enhancers/Enhancers.vue [hunk 1]
```diff
@@ -55,15 +56,21 @@ export default Vue.extend({
...
     route() {
       this.openEmoji()
+      this.resetSearch()
     },
     'ui.enhancers.show'(value) {
-      if (value) this.openEmoji()
+      if (value) {
+        this.openEmoji()
+      } else {
+        this.resetSearch()
```
Comment (WanderingHogan) [inline components/views/chat/enhancers/Enhancers.vue:72]: Can you use a guard clause here, eg:

if (value) {
  this.openEmoji()
  return
}
this.resetSearch()

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
ios is forcing capitalized first letter input, which is case sensitive. If you force lowercase on the v-model data it should fix Phils issue:

template: `<emoji-picker @emoji="addEmoji" :search="search.toLowerCase()">`

File: components/views/chat/enhancers/Enhancers.vue (hunk 1 above)
Comment (AlexZakablukov) [reply components/views/chat/enhancers/Enhancers.vue:72]: ok, thx)

**Comment by AlexZakablukov** [comment]
> ios is forcing capitalized first letter input, which is case sensitive. If you force lowercase on the v-model data it should fix Phils issue:
> 
> template: `<emoji-picker @emoji="addEmoji" :search="search.toLowerCase()">`

thank you, fixed)

**Review by WanderingHogan — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**